config/database.yml

```diff
@@ -1,25 +1,22 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
-default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
-
 development:
-  <<: *default
-  database: db/development.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: smartflix_development
+  pool: 5
+  host: localhost
+#  username: postgres_user_name
+#  password: postgres_password
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: db/test.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: smartflix_test
+  pool: 5
+#  username: postgres_user_name
+#  password: postgres_password
+
+staging:
+  url: <%= ENV['DATABASE_URL'] %>
 
 production:
-  <<: *default
-  database: db/production.sqlite3
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
I have a few suggestions for this file, perhaps you can create a default group so you don't need to repeat the same keys in all envs, I also would remove the staging env, since staging generally we run as rails env production, otherwise, we'll need to create more configs for this env, which might cause a different env, hence some error might just happen in prod. Also perhaps using utf8 instead of unicode as encoding, at least is what I generally use, but feel free to dismiss this 😄 
And last but not least I'd load the username and password as an ENV, something like this:


```yml
default: &default
  adapter: postgresql
  encoding: unicode
  pool: 5
  host: <%= ENV.fetch('DB_HOST') { 'localhost' } %>
  username: <%= ENV['DB_USER'] %>
  password: <%= ENV['DB_PASSWORD'] %>

development:
  <<: *default
  database: smartflix_development

test:
  <<: *default
  database: smartflix_test

production:
  <<: *default
  database: smartflix_production
```